### PR TITLE
[YLI-454]: Fix image media entity thumbnail update issue.

### PR DESCRIPTION
**Pull request by fandm**
Make sure these boxes are checked before asking for review of your pull request - thank you!

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [ ] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

## Issue
https://github.com/ymcatwincities/openy/issues/801
## Steps for review
- [ ] Login as admin.
- [ ] Go to the page /admin/content/media.
- [ ] Edit some Image media entity and change the image. Save. After return to the page /admin/content/media you will see the updated thumbnail.

Thank you.

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2253 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2253 |
| Installation process | http://install.openy-dev.ffwua.com/build2253/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2253/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2253/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2253/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2253/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2253/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2253/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build2253/profiles/contrib/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

File: modules/openy_features/openy_media/openy_media.module [hunk 1]
```diff
@@ -63,3 +64,14 @@ function openy_media_preprocess_entity_embed_container(array &$variables) {
...
   ]);
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ *
+ * Please see https://www.drupal.org/node/2749695#comment-11950133.
+ */
+function openy_media_media_presave(MediaInterface $media) {
+  if ('image' == $media->bundle() && !$media->isNew()) {
```
Comment (alexschedrov) [inline modules/openy_features/openy_media/openy_media.module]: This should be places in `openy_media_image` module.

Comment (alexschedrov) [reply modules/openy_features/openy_media/openy_media.module]: According to standards conditions should be reversed `$media->bundle() == 'image'`.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2256 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2256 |
| Installation process | http://install.openy-dev.ffwua.com/build2256/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2256/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2256/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2256/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2256/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2256/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2256/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build2256/profiles/contrib/openy/build/reports/behat

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2260 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2260 |
| Installation process | http://install.openy-dev.ffwua.com/build2260/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2260/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2260/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2260/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2260/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2260/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2260/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2261 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2261 |
| Installation process | http://install.openy-dev.ffwua.com/build2261/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2261/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2261/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2261/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2261/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2261/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2261/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build2260/profiles/contrib/openy/build/reports/behat

File: modules/openy_features/openy_media/openy_media.module (hunk 1 above)
Comment (fandm) [reply modules/openy_features/openy_media/openy_media.module]: Done.

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build2261/profiles/contrib/openy/build/reports/behat

**Review by ddrozdik — APPROVED** [review]

**Pull request merged**